edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -38,8 +38,44 @@
         You will be asked to verify your identity before you begin the exam. Make sure you have valid photo identification, such as a driver's license or passport, before you continue.
       {% endblocktrans %}
     </p>
+    <p>
+        {% blocktrans %}
+          When you have finished setting up the proctoring application, you may start the exam.
```

Comment:
@dianakhuang There's a lot more space below the "Start Proctored Exam" button than above it. Not critical, obviously, but adding a bit more space above the button and reducing the space below might improve appearances.